client/src/main/java/org/eclipse/hono/client/ApplicationClientFactory.java

```diff
@@ -67,11 +90,32 @@ static ApplicationClientFactory create(final HonoConnection connection, final Se
             Consumer<Message> telemetryConsumer,
             Handler<Void> closeHandler);
 
+    /**
+     * Creates a client for consuming events from Hono's north bound <em>Event API</em>.
+     *
+     * @param tenantId The tenant to consume events for.
+     * @param eventConsumer The handler to invoke with every event received.
+     * @param closeHandler The handler invoked when the peer detaches the link.
+     * @param autoAccept {@code true} if received deliveries should be automatically accepted (and settled)
```

Comment:
why would you want to set this to false?

Reply:
See above.